domain/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <kotlin.version>1.3.21</kotlin.version>
+        <kotlin.version>1.3.50</kotlin.version>
```

Comment:
I guess it would be better to specify a common versions like this one in one place for the re-usability i.e. a main `pom.xml` which specifies modules as well

Reply:
There is a task for merging domain and adapters modules for simplicity sake

Reply:
👍